tests/login.test.js

```diff
@@ -0,0 +1,88 @@
+describe('DigitalBank', () => {
+
+    //example test
+    it('Should enter the site', async () => {
+      browser.url('login');
```

Comment:
This test needs an assertion, as in its current state, it's not really validating anything from the page, except commanding the webdriver to browse a given page.

Add an assertion that checks for a condition that helps the test figure out if expected conditions are being met.
Eg.

```javascript
browser.url('login');
expect(browser.title().value()).toMatch('login-page');
```

This last **expect** line will fail the test if that title is not found on the page it's browsing (That is an example of an element you can look at to determine success of an assertion, you can check for any other element that signals that you are browsing the right page)